label_studio/ml/api.py

```diff
@@ -40,14 +41,24 @@ def perform_create(self, serializer):
         ml_backend.update_state()
 
 
+@method_decorator(name='perform_update', decorator=swagger_auto_schema(
+    tags=['Machine Learning'],
+    operation_summary='Add ML Backend',
+    operation_description="""
+    Add an ML backend using the Label Studio UI or by sending a POST request using the following cURL command:
+    ```bash
+    curl -X POST {host}/api/ml -H 'Authorization: Token abc123'\\
+    --data '{"url": "http://localhost:9090", "project": {{project_id}}}' 
+    """.format(host=(settings.HOSTNAME or 'https://localhost:8080')),
+))
```

Comment:
@makseq Please check this!

Reply:
@smoreface looks like we can add a token as var too: 

```
curl -X POST {host}/api/ml -H 'Authorization: Token {token}'\\ .... .format(host=..., token='abc123')
```

Reply:
@makseq I agree, but I only want to do that if we can pull in the customer token dynamically like we're doing for the hostname. Is that stored somewhere we can access or not?